admin/schroot/Makefile

```diff
@@ -0,0 +1,84 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=schroot
+PKG_VERSION:=1.6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.xz
+PKG_SOURCE_URL:=http://deb.debian.org/debian/pool/main/s/schroot
+PKG_HASH:=3ce8dfd9cb97b099e4b6d4ccec421d6cc8c9ef84574681e928a12badb5643d0b
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/schroot
+  SECTION:=admin
+  CATEGORY:=Administration
+  TITLE:=Securely enter a chroot and run a command or login shell.
+  DEPENDS:=$(ICONV_DEPENDS) @!mips \
```

Comment:
Why @!mips ?

Reply:
Because it breaks the CI on mips without it. I tried using `PKG_USE_MIPS16:=0` but it broke too.

Reply:
Ohhhh this breaks because of boost. This will not happen on the buildbots. It's a RAM problem with CircleCI (256MB I think).

I compiled this locally just fine. With MIPS16 as well.

Reply:
What shall I do then, put back PKG_USE_MIPS16:=0 ? Or not even that is needed?

Reply:
That;s not needed either. Just remove @!mips and let it fail.

Reply:
> That;s not needed either. Just remove @!mips and let it fail.

Done.